docs/CreateAKSClusterForScalarDB.md

```diff
@@ -0,0 +1,156 @@
+# Create an AKS cluster for ScalarDB Server
```

Comment:
Very sorry for the late review.
For this, can it be converted for ScalarDB Cluster?
Or, we should keep it but mark it as deprecated?

Also, I feel the tile should be something like this since users don't create a cluster by following the document.
Thought?
```
A guideline for creating an AKS cluster for ScalarDB Server
```

Reply:
Thank you for your review!

> For this, can it be converted for ScalarDB Cluster?
> Or, we should keep it but mark it as deprecated?

I will create a new document for ScalarDB Cluster in another PR.
And, in that PR, I will make these ScalarDB Server documents deprecated.

> Also, I feel the tile should be something like this since users don't create a cluster by following the document.
> Thought?

Ah, I see. Indeed, these documents don't describe `how to create managed k8s cluster`. These documents describe only the requirements and recommendations.

I agree with your suggestion. I updated all documents in 2a63f33145871a88697194f734005123c3a551b3.

Note: We have to update link (document name) in other documents, but we are updating those related documents (i.e., it might cause conflict). So, I will update those links in another PR after related PRs will be merged.